docs/guided-tour/import-export/export-parameters/README.md

```diff
@@ -0,0 +1,210 @@
+the default of 5 minutes.
+
+Note that the mechanism described here is specific for the [helm deployer][6] and [manifest deployer][7].
+
+
+### Export Executions
+
+In the previous step, we have read values from the target cluster. These are not yet the export values of the
+Blueprint. The actual export values of the Blueprint are constructed by a template specified in
+section `exportExecutions` of the [Blueprint](./blueprint/blueprint.yaml). 
+The template should produce a yaml of the following structure: 
+
+```yaml
+exports:
+  <parameter name>: <parameter value>
+  <parameter name>: <parameter value>
+  ...
+```
+
+In our case, the template is:
+
+```yaml
+exports:
+  token: {{ index .deployitems "default-deploy-item" "test-token" | b64dec }}
+```
+
+The data that can be filled into the template are the values that the `DeployItems` have read 
+from the target cluster. Besides them, also other data like the imports of the Blueprint can be used in the template.
+All these data are organized in a tree:  
+
+```shell
+.
+├── deployitems
+│   └── default-deploy-item  # name of the DeployItem
+│       └── test-token       # key of the value provided by the DeployItem
+├── imports
+│   └── ...
+...
+```
+
+Normally, you can access a field of the above tree by a path like 
+`.deployitems.<name of deployitem>.<key>`. However, if the path contains "special" characters like a hyphen, you must 
+use the `index` function instead: `index .deployitems "<name of deployitem>" "<key>"`.
+
+For details, see [Blueprints: Rendering][4]
+
+
+## Where an Export Goes To
+
+An `Installation` must define what to do with the exports of its Blueprint. This is done in the section `exports` of 
+the `Installation`. It connects each export parameters of the blueprint with a corresponding `DataObject` or `Target` 
+where the value will be stored. 
+These objects will be created in the same namespace as the `Installation`.
+
+The `DataObjects` and `Targets` that one `Installation` exports can be imported by another `Installation`.
+
+In our case, the export parameter `token` will be written to a `DataObject` with name `my-token`: 
+
+```yaml
+exports:
+  data:
+    - name: token        # name of the export parameter of the blueprint
+      dataRef: my-token  # name of the DataObject
+```
+
+The general structure of the `exports` section of an `Installation` is this:
+
+```yaml
+exports:
+  targets:
+    - name:    <name of the export parameter of the blueprint>
+      target:  <name of a Target in which the value will be stored>
+  data:
+    - name:    <name of the export parameter of the blueprint>
+      dataRef: <name of a DataObject in which the value will be stored>
+```
+
+For more details, see [Installations: Exports][5].
+
+
+The following diagram shows the linkages between Blueprint, Installation and DataObject, that we have discussed:
+
+![linkages](./linkages.png)
+
+
+## Procedure
+
```

Comment:
Not sure but perhaps we should add one sentence that we now start with the example? Perhaps users will read this stand alone as I and might be lost after the long (but very helpful) introduction.

Reply:
Done